app/routes/admin/sales/revenue.js

```diff
@@ -0,0 +1,14 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+  titleToken() {
+    return this.l10n.t('Revenue');
+  },
+
+  async model() {
+    let orders = await this.store.findAll('admin-sales-fee', { reload: true }).then(orders => orders.sortBy('name'));
+    let events = await this.store.findAll('event', { reload: true }).then(events => events.sortBy('name'));
+    orders.forEach((data, index) => data.set('eventDate', events[index].startsAtDate));
```

Comment:
Please do not assume the indexes of both orders and events match. (ie) for example, do not assume `orders[0]` matches with `events[0]`

Reply:
I actually assumed it on the basis that I sorted them on the basis of name. I guess name in orders and events are event name only, right? I wasn't able to find another way to get event date as their is no relation to event in orders model. Maybe we can discuss this in tomorrow's meeting.